src/models/Token/index.js

```diff
@@ -1,17 +1,53 @@
 // @flow
-import createToken from './createToken'
-import retrieveToken from './retrieveToken'
-import validateToken from './validateToken'
-export * from './createToken'
-
-export type Token = {
-  id: string,
-  token: string,
-  userId: string
-}
+import { Model } from 'objection'
+import User from '@models/User'
+
+class Token extends Model {
+  $beforeInsert() {
+    this.createdAt = new Date().toISOString()
```

Comment:
Why to ISO here but not in the Project class? Let’s remove it if we don‘t need it